OpenEdXMobile/res/values/styles.xml

```diff
@@ -582,15 +582,24 @@
         <item name="lineHeight">@dimen/widget_margin_double</item>
     </style>
 
-    <style name="EmptyStateButton" parent="edX.Widget.ActionBrandPillButton">
+    <style name="CourseDashboardHeadline">
+        <item name="android:textColor">@color/neutralBlackT</item>
+        <item name="android:textSize">@dimen/edx_large</item>
+        <item name="android:gravity">start</item>
+        <item name="lineHeight">@dimen/widget_margin_x_double</item>
+    </style>
+
+    <style name="PrimaryActionButton">
```

Comment:
`edX.Widget.PrimaryActionButton` is already defined, better to override.

Reply:
that button will be removed in app_nav